cnc_ctrl_v1/CNC_Functions.h

```diff
@@ -1315,40 +1315,48 @@ void  interpretCommandString(const String& cmdString){
     /*
     
     Splits a string into lines of gcode which begin with 'G'
+
+    Assumptions:
+        Leading and trailing white space has already been removed from cmdString
```

Comment:
❤️ Thanks for adding comments that clarify things that weren't clear as you were going through the code, that's always SUPER helpful because you've got fresh eyes